pkgs/applications/networking/feedreaders/feed2imap-go/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "feed2imap-go";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "Necoro";
+    repo = "feed2imap-go";
+    rev = "v${version}";
+    sha256 = "sha256-Ywo+ll6Ysy7H3bfzTBsT8U7M9XVOBlfd0v7RMgKoitY=";
+  };
+
+  vendorHash = "sha256-dtui0eChZXTrvprX3py/xCGmbHUYhk7nCHQ9pwBMHRY=";
+
+  postInstall = ''
+    # Move the print-cache tool out of the PATH, seems to be an internal development tool
+    mkdir $out/libexec
+    mv $out/bin/print-cache $out/libexec
```

Comment:
Is this even used or does it work in this location without extra patches? if not we can just delete it

Reply:
Good question. I assumed it was a debugging tool; not worth having in the PATH, but good to have around. I’m asking upstream at https://github.com/Necoro/feed2imap-go/issues/94

Reply:
Resolved, the binary should be deleted. Updated and squashed this branch.